feature-libs/cart/base/core/store/actions/cart-voucher.action.ts

```diff
@@ -42,6 +45,7 @@ export class CartAddVoucherFail extends StateUtils.EntityFailAction {
     }
   ) {
     super(PROCESS_FEATURE, ADD_VOUCHER_PROCESS_ID, payload.error);
+    this.error = payload.error;
```

Comment:
[question]
Do we really need this line? I can see we don't need it e.g. in `CreateWishListFail`. But both those actions `extends StateUtils.EntityFailAction` - which does the job of assigning value to the public `error` property.

Note: analogical question applies to other places in this PR

Reply:
It's a leftover. I made this change before modifying `EntityFailAction`.